plugins/kubernetes/README.md

```diff
@@ -108,14 +108,14 @@ Kubernetes::Release
 
 ### Docker Images
 
-(To opt out of this feature set `containers[].samson/dockerfile: none` or `metadata.annotations.container-nameofcontainer-samson/dockerfile: none`)
+(To opt out of this feature set `metadata.annotations.container-nameofcontainer-samson/dockerfile: none`)
```

Comment:
this annotation makes me sad because it's so long and wordy, would have preferred to have seen JSON or YAML inside the annotation instead